### PR TITLE
Allow comma in terms

### PR DESCRIPTION
**Pull request by annyan09023**
This change is to allow comma in terms. The unit test was changed by adding term `apple,123`

### CODE REVIEW

File: avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_2/fruits_day2 [hunk 1]
```diff
@@ -2,5 +2,6 @@ tue_fruits,pear
 tue_fruits,apple
 tue_fruits,banana
 tue_fruits,kiwi
+tue_fruits,apple,123
```
Comment (haitong) [inline avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_2/fruits_day2]: Please also add some special characters like @ $ . % ... to terms and make sure it works as expected

File: avro2tf/src/test/scala/com/linkedin/avro2tf/jobs/Avro2TFTest.scala [hunk 2]
```diff
@@ -3,6 +3,7 @@ package com.linkedin.avro2tf.jobs
 import java.io.{File, FileOutputStream, PrintWriter}
 
 import com.linkedin.avro2tf.constants.Avro2TFJobParamNames
+import com.linkedin.avro2tf.constants.Constants.SPLIT_REGEX
```
Comment (haitong) [inline avro2tf/src/test/scala/com/linkedin/avro2tf/jobs/Avro2TFTest.scala]: Is this import used?

File: avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_1/fruits_day1 [hunk 3]
```diff
@@ -1,6 +1,7 @@
-mon_fruits,apple
 mon_fruits,pear
+mon_fruits,apple
 mon_fruits,banana
 mon_fruits,kiwi
+mon_fruits,apple,123
 mon_fruits,grapes
-mon_fruits,pineapple
+mon_fruits,pineapple
```
Comment (haitong) [inline avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_1/fruits_day1:7]: interesting as there is no change but git is showing adding/deleting lines here.

File: avro2tf/src/test/scala/com/linkedin/avro2tf/jobs/Avro2TFTest.scala (hunk 2 above)
Comment (annyan09023) [reply avro2tf/src/test/scala/com/linkedin/avro2tf/jobs/Avro2TFTest.scala]: yes, already removed in latest commit

**Review by zhangxuhong — APPROVED** [review]

File: avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_2/fruits_day2 (hunk 1 above)
Comment (annyan09023) [reply avro2tf/src/test/resources/data/share_feat_list/feat_lists_case_2/fruits_day2]: done. Thanks

**Pull request merged**